tests/testthat/test-post_tweet.R

```diff
@@ -0,0 +1,8 @@
+test_that("post_tweet works", {
+  skip_if_offline()
+  skip_on_cran()
+  
+  rt <- expect_message(post_tweet("test"), "your tweet has been posted!")
```

Comment:
I just realised this is probably a bad idea — I don't really want to be sending tweets from my twitter account just by running the tests. We should skip this until we have a twitter account just for testing.

Reply:
You got funny replies :D But that means that post_destroy didn't work well, so it was a good test. Anyway this also highlights the need of using/managing different tokens